include/nrsc5.h

```diff
@@ -263,6 +263,22 @@ struct nrsc5_sis_dsd_t
  */
 typedef struct nrsc5_sis_dsd_t nrsc5_sis_dsd_t;
 
+/**
+* ID3 comment descriptor. This is a linked list element that may point to further
+* comments via `next` member if not `NULL`.
+* Refer to HD-Radio document SY_IDD_1028s.
+*/
+struct nrsc5_id3_comm {
+    struct nrsc5_id3_comm *next; /**< Pointer to next element or NULL */
+    char *lang; /**< language code, e.g. "eng" */
+    char *short_content; /**< short content description */
+    char *actual_text; /**< actual text */
+};
+/**
+ * Defines a typename for struct nrsc5_id3_comm_tag
+ */
+typedef struct nrsc5_id3_comm nrsc5_id3_comm;
```

Comment:
All other structs have a `_t` suffix, so we should probably use that here for consistency.

Also, it might be clearer to use the full `comment` instead of just `comm`.